scout/constants/case_tags.py

```diff
@@ -57,7 +57,7 @@
 CANCER_PHENOTYPE_MAP = {1: "normal", 2: "tumor", 0: "unknown", -9: "unknown"}
 REV_PHENOTYPE_MAP = {value: key for key, value in PHENOTYPE_MAP.items()}
 
-CASE_STATUSES = ("prioritized", "inactive", "active", "solved", "archived")
+CASE_STATUSES = ("prioritized", "inactive", "active", "solved", "archived", "unused")
```

Comment:
I don't like much 'discrepant' and 'discordant' because these terms imply a non-specified category assumed to be normal (discrepant from what?). 'Extraneous' or 'extrinsic' are not self explanatory at all, and might be more confusing than useful for the institutes that didn't ask for the feature. 'Other' is vague, but perhaps is OK since everybody can attribute it their own meaning. Disregarded could also be OK perhaps.

Could be also 'ignored',  'unprocessed', 'abandoned' ?

What I like most is `abandoned` and `other`. What about your preferences?

Reply:
Im not so fond of abandoned - they typically aren't, only not tallied, and if they were it seems like a call to go rescue them. 😅

Other works well right now, but I have a gnawing feeling we may regret that later when several user groups have their own interpretation of it, and want new features connected to it.

Ignored is better! I still like the "extra" line more, but I can easily see that as non-obvious as well.

Reply:
So, ignored?